bn_s_mp_div_small.c

```diff
@@ -0,0 +1,72 @@
+#include "tommath_private.h"
+#ifdef BN_S_MP_DIV_SMALL_C
+/* LibTomMath, multiple-precision integer library -- Tom St Denis */
+/* SPDX-License-Identifier: Unlicense */
+
+#ifdef BN_MP_DIV_SMALL
```

Comment:
Remove old guard

Reply:
Just two weeks before he is eligible to his full pension?
Ow, c'mon, really?